utils/file.js

```diff
@@ -76,14 +78,16 @@ exports.fileSystem = (function(){
 
       while (null !== (chunk = readable.read(chunkSize))) {
         const chunkId = sha1HashData(chunk);
-        manifest.chunks.push(chunkId);
+        manifest.chunks[chunkId] = [];
 
+        copyShards(chunk, chunkId, manifest)
         storeShards(chunk, chunkId)
       }
     });
 
     readable.on('end', () => {
-      fileSystem.writeFile(`${dir}/${manifestName}`, JSON.stringify(manifest), () => {
+      // fileSystem.writeFile(`${dir}/${manifestName}`, JSON.stringify(manifest), () => {
```

Comment:
I think we could probably remove this commented out line of code. Might be nice to have an explanation of what '\t' is doing here.